controller_interface/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ find_package(catkin REQUIRED COMPONENTS roscpp hardware_interface pluginlib)
 
 # Declare catkin package
 catkin_package(
-  DEPENDS boost
```

Comment:
However, if `Boost_INCLUDE_DIRS` and `Boost_LIBRARIES` aren't used in the `CMakeLists.txt`, I guess it's because nothing really depends on `Boost` here, right? Indeed, `find_package(Boost REQUIRED)` is needed as well, if `Boost` is actually used.